action/src/main/java/com/adyen/checkout/action/DefaultActionHandlingComponent.kt

```diff
@@ -32,6 +32,9 @@ class DefaultActionHandlingComponent(
     override fun handleAction(action: Action, activity: Activity) {
         activeDelegate = genericActionDelegate
```

Comment:
Should this be removed?

Reply:
Nope, the `PaymentInProgressView` will crash if we'd do that